packages/berry-fslib/sources/NodeFS.ts

```diff
@@ -226,11 +231,38 @@ export class NodeFS extends FakeFS {
     };
   }
 
-  private fromPortablePath(p: string) {
-    return p;
+  static fromPortablePath(p: string) {
+    if (process.platform !== `win32`) return p;
+
+    // Path should look like "/mnt/n/berry/scripts/plugin-pack.js"
+    // And transform to "N:\berry/scripts/plugin-pack.js"
+
+    if (!PORTABLE_PREFIX_REGEXP.test(p)) return p;
+
+    const driveLetter = p[PORTABLE_PATH_PREFIX.length].toUpperCase();
+    const pathWithoutPrefix = p.substr(PORTABLE_PATH_PREFIX.length + 1);
+    const windowsPath = pathWithoutPrefix.replace(/\//g, '\\\\');
```

Comment:
Shouldn't it be `'\\'` (otherwise it turns into double backslashes)?

Reply:
Yes, I changed it, I tried some things to make it work, forgot to revert it 😅 
(It actually also work with it)